bindings/wasm/examples/model-viewer.html

```diff
@@ -159,8 +165,25 @@
     simplified.computeVertexNormals();
     return simplified;
   }
+
+  require.config({ paths: { vs: 'https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.34.0/min/vs' } });
```

Comment:
Why did you end up going with monaco instead of code-mirror? I ask because I never saw a response to https://github.com/elalish/manifold/issues/73#issuecomment-1238438742

Reply:
Because monaco supports typescript and language server out of the box.